docs/content/config/environment.md

```diff
@@ -13,13 +13,6 @@ title: Environment Variables
 - **empty** => uses the `hostname` command to get canonical hostname for `docker-mailserver` to use.
 - => Specify a fully-qualified domainname to serve mail for. This is used for many of the config features so if you can't set your hostname (_eg: you're in a container platform that doesn't let you_) specify it via this environment variable. It will take priority over `docker run` options: `--hostname` and `--domainname`, or `docker-compose.yml` config equivalents: `hostname:` and `domainname:`.
 
-##### DMS_DEBUG
-
-This environment variable is deprecated. Use `LOG_LEVEL` instead.
-
-- **0** => Debug disabled
-- 1     => Enables debug on startup
-
```

Comment:
As the old var is referenced in a huge amount of discussions and issue comments we should think about keeping it in here for a few more releases. "is deprecated" should be "was removed" as deprecated would mean it is still in place.

Reply:
That's a good point! :)

I think I've had that confusing experience as a user before a few times :sweat_smile:

Reply:
Good point, I will take care :D

Reply:
Done in 05f483554433eec6beadccc076a5432f8e262d38